app/javascript/controllers/metadata_template_dd_controller.js

```diff
@@ -57,6 +56,145 @@ export default class extends Controller {
     }
   }
 
+  menuTargetConnected(element) {
+    element.addEventListener("keydown", this.boundOnMenuItemKeyDown);
+    element.addEventListener("focusout", this.boundFocusOut);
```

Comment:
In the `dropdown_controller.js` you set the tabindex to all items to -1 in here.  Should we also do that here?